css/MLS.css

```diff
@@ -1,4 +1,4 @@
-/* This CSS makes adjustments for the MLS on top of the style sheets that come with LaTeXML.
+﻿/* This CSS makes adjustments for the MLS on top of the style sheets that come with LaTeXML.
```

Comment:
Is this some strange white-space change?  I can't see the difference.

Reply:
It's related to the other mistake.

Reply:
And should be gone now.

Reply:
Yes.  Resolving.